oauth2client/flask_util.py

```diff
@@ -191,6 +192,21 @@ def requires_calendar():
 __author__ = 'jonwayne@google.com (Jon Wayne Parrott)'
 
 DEFAULT_SCOPES = ('email',)
+CREDENTIALS_KEY = 'google_oauth2_credentials'
```

Comment:
Let's be conservative and not support use of these constants outside of this module. If someone wants to violate our API and use them of course the interpreter will let them and if someone is so happy with how that turns out that they want formal support they can file an issue and explain the situation and add test coverage with their patch and we can go through the whole process.